optaweb-vehicle-routing-docs/src/main/asciidoc/run-locally.adoc

```diff
@@ -43,6 +47,14 @@ You can use the interactive mode to download additional OSM files from Geofabrik
 . Repeat the previous step until you see a list with the region you want to download.
 . Download a region by entering its ID and then entering `d`.
 
+[WARNING]
+.Using large OSM files
+====
+For best user experience, use smaller regions like individual European or US states.
```

Comment:
For the best user...regions such as...

(Usually 'such as' is better than 'like')